database/schema.sql

```diff
@@ -103,10 +103,16 @@ CREATE TABLE public.organization_members (
 CREATE TABLE public.jobs (
   id UUID PRIMARY KEY NOT NULL DEFAULT gen_random_uuid(),
   organization_id UUID NOT NULL REFERENCES public.organizations (id),
-  title VARCHAR NOT NULL CHECK (title <> ''),
-  description TEXT NOT NULL CHECK (description <> ''),
-  position VARCHAR CHECK (position <> ''),
-  opportunity VARCHAR CHECK (opportunity <> ''),
+  jobtitle VARCHAR NOT NULL CHECK (jobtitle <> ''),
+  jobdescription TEXT NOT NULL CHECK (jobdescription <> ''),
+  jobposition VARCHAR CHECK (jobposition <> ''),
+  jobtype VARCHAR CHECK (jobtype <> ''),
+  jobcompensation VARCHAR CHECK (jobcompensation <> ''),
+  jobmax VARCHAR CHECK (jobmax <> ''),
+  jobmin VARCHAR CHECK (jobmin <> ''),
+  jobequity VARCHAR CHECK (jobequity <> ''),
+  joblocation VARCHAR CHECK (joblocation <> ''),
+  jobcontact VARCHAR CHECK (jobcontact <> ''),
```

Comment:
Having a prefix of "job" is too redundant, we can omit that. We already have the job context coming from the table name. And in terms of TypeScript typings, we will always have the job context. So it's safe to omit the "job" prefix for them attributes.